pkg/component/nodeproblemdetector/node_problem_detector.go

```diff
@@ -226,7 +225,7 @@ func (c *nodeProblemDetector) computeResourcesData() (map[string][]byte, error)
 								Command: []string{
 									"/bin/sh",
 									"-c",
-									"exec /node-problem-detector --logtostderr --config.system-log-monitor=/config/kernel-monitor.json,/config/docker-monitor.json,/config/systemd-monitor.json .. --config.custom-plugin-monitor=/config/kernel-monitor-counter.json,/config/systemd-monitor-counter.json .. --config.system-stats-monitor=/config/system-stats-monitor.json --prometheus-address=" + daemonSetPrometheusAddress + " --prometheus-port=" + strconv.Itoa(daemonSetPrometheusPort),
```

Comment:
According to https://github.com/kubernetes/node-problem-detector/blob/34b265af347190b4dff320d77711bc92a5e8342d/cmd/options/options.go#L126 it looks like not setting this option will default to `127.0.0.1`. Is this what we want here?

Reply:
Technically not but we noticed that some applications work just fine with binding set "127.0.0.1" and some do not. That's why for some it's just removed and for some we have it explicitly

Reply:
Ok, it does not look like we use this endpoint, e.g. for metrics. Hence, this should be fine.